test/unit/lib/req-info.js

```diff
@@ -26,13 +26,13 @@ describe('lib/req-info.js', () => {
       });
 
       describe('with keycloak-gatekeeper headers', () => {
-        describe('with a single audience', () => {
+        describe('with a single audience using (short header) x-auth-aud', () => {
           let result;
 
           before(() => {
             result = reqInfo({
               headers: {
-                'x-auth-audience': 'client',
+                'x-auth-aud': 'client',
```

Comment:
I think we should have both tests - one for aud and one for audience

Reply:
yes there is another one "with multiple audiences" which uses full x-auth-audience